test/spec/Feature/Query/MultipleSchemaSpec.hs

```diff
@@ -111,7 +122,7 @@ spec =
         request methodPost "/children" [("Content-Profile", "unknown")]
           [json|{"name": "child 4", "parent_id": 4}|]
           `shouldRespondWith`
-          [json|{"message":"The schema must be one of the following: v1, v2","code":"PGRST106","details":null,"hint":null}|]
+          [json|{"message":"The schema must be one of the following: v1, v2, SPECIAL \"@/\\#~_-","code":"PGRST106","details":null,"hint":null}|]
```

Comment:
Oh, how would it look here if the `SPECIAL \"@/\\#~_-` schema had a comma(`,`)?

Reply:
I added the comma to the tests and worked without issues. But I think the problem is that the configuration file does not accept the comma in the value, because it uses it as a splitter character: https://github.com/PostgREST/postgrest/blob/870f7a39b077269faebdc32006e7c41acff79b4a/src/PostgREST/Config.hs#L375-L377
Or am I missing a way to send commas as part of the schema name?